ftl/core/card-templates.ftl

```diff
@@ -61,4 +61,5 @@ card-templates-restore-to-default = Restore to Default
 card-templates-restore-to-default-confirmation = This will reset all fields and templates in this notetype to their default
     values, removing any extra fields/templates and their content, and any custom styling. Do you wish to proceed?
 card-templates-restored-to-default = Notetype has been restored to its original state.
-
+card-templates-example = example
+card-templates-sample = sample
```

Comment:
Since the example is intended to show how comparison works, can we add an explanation for translators so that they pick a good example in their languages, instead of literally translating the strings?

Reply:
Super reasonable and I had thought the same, but I don't know exactly how to do it yet here with Fluent, since I never went to Pontoon to see how things are displayed there.

From [Fluent docs](https://projectfluent.org/fluent/guide/comments.html), I think that I should add a comment, but still not sure

Reply:
Okay, found the [correct docs](https://translating.ankiweb.net/anki/developers). Will change it

Reply:
Added them. Thanks for the push!